ditto/writers/opendss/write.py

```diff
@@ -1839,59 +1839,178 @@ def write_loads(self, model):
+            
+            vmin_txt = ""
+            if hasattr(i, "vmin") and i.vmin is not None:
+                vmin_txt = " Vminpu={vmin}".format(vmin=i.vmin)
+            
+            vmax_txt = ""
+            if hasattr(i, "vmax") and i.vmax is not None:
+                vmax_txt = " Vmaxpu={vmax}".format(vmax=i.vmax)
+            
+            if balanced_load:  # only need to define one New Load regardless of number of phases
+                
+                txt += "New Load." + i.name
+                txt += bus1_txt
+                # balanced_load case:
+                for phase_load in i.phase_loads:  # add the .phs1.phs2 etc
+                    # add all phases onto Bus1
+                    if hasattr(phase_load, "phase") and phase_load.phase is not None:
+                        txt += f".{self.phase_mapping(phase_load.phase)}"
+
+                    # handle Delta connections
+                    if i.connection_type == "D" and len(i.phase_loads) == 1:
+                            if self.phase_mapping(i.phase_loads[0].phase) == 1:
+                                txt += ".2"
+                            if self.phase_mapping(i.phase_loads[0].phase) == 2:
+                                txt += ".3"
+                            if self.phase_mapping(i.phase_loads[0].phase) == 3:
+                                txt += ".1"
 
-                # Connection type
-                if hasattr(i, "connection_type") and i.connection_type is not None:
-                    if i.connection_type == "Y":
-                        txt += " conn=wye"
-                    elif i.connection_type == "D":
-                        txt += " conn=delta"
+                # if i.connection_type=='Y':
+                txt += " Phases={N}".format(N=len(i.phase_loads))
+                # elif i.connection_type=='D' and len(i.phase_loads)==3:
+                #    fp.write(' Phases=3')
+                # elif i.connection_type=='D' and len(i.phase_loads)==2:
+                #    fp.write(' Phases=1')
 
-                # Connecting element
-                if (
-                    hasattr(i, "connecting_element")
-                    and i.connecting_element is not None
-                ):
-                    txt += " bus1={bus}".format(bus=re.sub('[^0-9a-zA-Z]+', '_', i.connecting_element))
-                    if hasattr(i, "phase_loads") and i.phase_loads is not None:
-                        for phase_load in i.phase_loads:
+                txt += nomimal_voltage_txt
+                
+                txt += " kW={P}".format(P=sum(kws))
+                
+                txt += " model={N}".format(N=i.phase_loads[0].model)
+
+                txt += conn_type_txt
+
+                txt += " kvar={Q}".format(Q=sum(kvars))
+
+                txt += vmin_txt
+
+                txt += vmax_txt
+
+                # positions (Not mapped)
+
+                for phase_load in i.phase_loads:
```

Comment:
Hmmm isn't the section below potentially duplicating the changes made above to txt? Should the section below be removed?

Reply:
This section already existed and it only applies to ZIP load models. It has been awhile but I think I just left this here because I was not sure where it should go. I have not used the ZIP model in OpenDSS. Are you familiar with how we should parse it for unbalanced loads? Maybe the `if` can be part of the `for phase_load in i.phase_loads` above ~line 1967